backend/molgenis-emx2-fairdatapoint/src/main/java/org/molgenis/emx2/fairdatapoint/FAIRDataPoint.java

```diff
@@ -46,27 +46,20 @@ public class FAIRDataPoint {
           entry("r3d", "http://www.re3data.org/schema/3-0#"),
           entry("lang", "http://lexvo.org/id/iso639-3/"));
 
-  /**
-   * Constructor
-   *
-   * @param request
-   * @param schemas
-   * @throws Exception
-   */
-  public FAIRDataPoint(Request request, Schema... schemas) {
+  public FAIRDataPoint(Context ctx, Schema... schemas) {
```

Comment:
For another time: fair data point should not need ctx but instead return something that the service can render
(in other words: this molgenis-emx2-fairdatapoint package should now know about javalin(